### ISSUE TITLE
Add `@build_bazel_rules_swift//swift:defs.bzl`

### ISSUE THREAD

**Issue by UebelAndre** [bazelbuild/rules_swift#1200]
Many other rules use this pattern but noticed this repo doesn't. Would it be reasonable to add this file so loads are predictable?

**Comment by keith** [comment]
no strong pref on adding this or not but note that `swift:swift.bzl` is the equivalent today

**Comment by UebelAndre** [comment]
Yeah, my rationale for opening this issue is that this doesn't follow the (seemingly common) pattern of other rule repos and incurs some burden of knowledge for developers introducing swift to their repo. I don't think I care what the pattern is I just want it to be consistent (at least for projects in `bazelbuild`).
